pkg/bucketeer/model/unknown_error_metrics_event.go

```diff
@@ -9,10 +11,10 @@ type UnknownErrorMetricsEvent struct {
 	Type   metricsDetailEventType `json:"@type,omitempty"`
 }
 
-func NewUnknownErrorMetricsEvent(tag string, api APIID) *UnknownErrorMetricsEvent {
+func NewUnknownErrorMetricsEvent(tag string, code int, api APIID) *UnknownErrorMetricsEvent {
 	return &UnknownErrorMetricsEvent{
 		APIID:  api,
-		Labels: map[string]string{"tag": tag},
+		Labels: map[string]string{"tag": tag, "response_code": strconv.Itoa(code)},
```

Comment:
We should also implement the `error_message`.

Reference:
https://github.com/bucketeer-io/ios-client-sdk/pull/68/files#diff-c3220166f7e54cc576f13734d3d0d8f3d60a8e543e2fd9a56e4b7ea100a0f643R400-R404

https://github.com/bucketeer-io/ios-client-sdk/pull/68/files#diff-21e1c84fd4bacc9267f4a4cd72b9f6a4a37dd6ab724b13ee0d18973da4b186b9R156-R158

Reply:
@cre8ivejp 
I will consult with you regarding the information that is the basis of the `error_message`.
I'm thinking of using `http.Response.Status` below, what do you think?

https://pkg.go.dev/net/http#Response.Status

Since the reference says "e.g. 200 OK", I think it can be applied to the error message.
Do you have any other good ideas?